.github/workflows/spellcheck.yml

```
@@ -67,8 +67,8 @@ jobs:
             OpenScPCA admin
             spelling
 
-      - name: Fail if there are spelling errors
-        if: steps.spell.outputs.error_count > 0
+      - name: Fail if there are spelling errors for non-scheduled jobs
+        if: github.event_name != 'schedule' && steps.spell.outputs.error_count > 0
```

Does this need to be:

```suggestion
        if: ${{ github.event_name != 'schedule' && steps.spell.outputs.error_count > 0 }}
```

Going off line 60 here.

---

The syntax for `if:` statements is special in that GHA assumes the content is an expression, so you don't need the wrapper. See the first note in: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions